src/cc_catalog_airflow/dags/provider_api_scripts/cleveland_museum_of_art.py

```diff
@@ -92,6 +94,7 @@ def _get_response(
 def _handle_response(
                     batch
                     ):
```

Comment:
The format of this param looks a bit odd here.

Reply:
Definitely! Thank you for catching this!